lib/ContextParser.ts

```diff
@@ -705,7 +707,11 @@ must be one of ${Util.CONTAINERS.join(', ')}`, ERROR_CODES.INVALID_CONTAINER_MAP
             external: !!contextIris[i] || options.external,
             parentContext: accContext.getContextRaw(),
             remoteContexts: contextIris[i] ? { ...remoteContexts, [contextIris[i]]: true } : remoteContexts,
-          })),
+          },
+          // @ts-expect-error: This third argument causes a type error because we have hidden it from consumers
```

Comment:
I'm probably just blind, but what exactly causes this third arg to be hidden?

Reply:
See https://github.com/rubensworks/jsonld-context-parser.js/pull/73#discussion_r1377275517